src/content/contributing/adding-layer-2/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Adding Layer 2s
+description: The policy we use when adding a layer 2 to ethereum.org
+lang: en
+sidebar: true
+---
+
+# Adding layer 2s {#adding-layer-2}
+
+We want to make sure we list the best resources possible so users can navigate the layer 2 space in a safe and confident manner.
+
+Anyone is free to suggest adding a layer 2 on ethereum.org. If there's a layer 2 that we have missed, **please suggest it!**
+
+We currently list L2s on the following pages:
+
+- [Optimistic rollups](/developers/docs/scaling/optimistic-rollups/)
+- [Zero-knowledge rollups](/developers/docs/scaling/zk-rollups/)
+<!-- TODO: Add layer-2 page -->
+
+## How we decide {#how-we-decide}
+
+Layer 2 submissions will be assessed by the following criteria:
+
+**[Listing on L2BEAT](https://l2beat.com)**
+
+- In order to be considered, this project to be featured on L2BEAT. This resource provides a great risk assessment of layer 2 projects. **If the project is not featured on L2BEAT, they will not be considered for ethereum.org.**
+
+[Learn how to add your L2 project to L2BEAT](https://github.com/l2beat/l2beat/blob/master/CONTRIBUTING.md).
+
+**Is the project open source?**
+
+**What kind of layer 2 is it?**
+
+- Optimistic rollup
+- Zero-knowledge rollup
+- State channel
+
+**Bridges**
+
+- How are users able to onboard to the layer 2?
+
+**Exchange support**
+
+- Are users able to deposit and/or withdraw directly from an exchange? If yes, which exchanges?
+
+**Links to dapps in the layer 2 ecosystem**
+
+- We want to be able to provide information on what users can expect to be able to do on this layer 2.
+
+**Token contract lists**
+
+- Since assets will have a new address on layer 2, if there is a token list resource available please share.
+
+**How long has the layer 2 been live?**
+
+**Is the project a fork?**
+
+- If the layer 2 is a fork, what project is it a fork of?
+
+**What wallets can users use to access this layer 2?**
+
+- What is the public RPC information for the network? (RPC URL, Chain ID, Currency Symbol, Block Explorer URL)
+- What wallets natively support this layer 2?
+
+**Has the project been externally audited?**
+
+- Include links to report and/or repo.
+
+**Is the layer 2 widely used?**
+
+- We will consider metrics such as TVL history, transaction statistics, and whether it is used by known companies or projects
+
+**Is the project being actively maintained?**
+
+- We won't list layer 2s that don't have an active team supporting the project.
+
+**Is there a block explorer?**
```

Comment:
Adding this in case someone answers 'yes' without providing a link 😅

```suggestion
**Is there a block explorer?**

- Please provide a link to the block explorer.
```